gostream/codec/h264/ffmpeg/avcodec/avcodec.go

```diff
@@ -0,0 +1,272 @@
+// Package avcodec is a thin wrapper around FFmpeg/release4.4.
+// See: https://github.com/FFmpeg/FFmpeg/tree/release/4.4
+package avcodec
+
+//#cgo CFLAGS: -Wno-deprecated-declarations -I${SRCDIR}/../ffmpeg/include
+//#cgo LDFLAGS: -L${SRCDIR}/../ffmpeg/lib -lavformat -lavcodec -lavutil -lavdevice -lavfilter -lswresample -lswscale -lm -lX11
```

Comment:
```suggestion
//#cgo LDFLAGS: -L${SRCDIR}/../ffmpeg/lib -lavformat -lavcodec -lavutil
```
As with avutil.go I think you're including a lot of unused libraries.

Reply:
Done.